File: root/files/resource/Sql/sqlserver/Automatic generation/DTO/TerritoriesEntity.vb
'**********************************************************************************
'* クラス名        ：TerritoriesEntity
'* クラス日本語名  ：自動生成Entityクラス
'*
'* 作成日時        ：2014/2/9
'* 作成者          ：棟梁 D層自動生成ツール（墨壺）, 日立 太郎
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'**********************************************************************************

' System～
Imports System

''' <summary>自動生成Entityクラス</summary>
<Serializable> _
Public Class TerritoriesEntity
	#Region "メンバ変数"

	''' <summary>設定フラグ：TerritoryID</summary>
	Public IsSetPK_TerritoryID As Boolean = False

	''' <summary>メンバ変数：TerritoryID</summary>
	Private _PK_TerritoryID As System.String

	''' <summary>プロパティ：TerritoryID</summary>
	Public Property PK_TerritoryID() As System.String
		Get
			Return Me._PK_TerritoryID
		End Get
		Set
			Me.IsSetPK_TerritoryID = True
			Me._PK_TerritoryID = value
		End Set
	End Property

	''' <summary>設定フラグ：TerritoryDescription</summary>
	Public IsSet_TerritoryDescription As Boolean = False

	''' <summary>メンバ変数：TerritoryDescription</summary>
	Private _TerritoryDescription As System.String

	''' <summary>プロパティ：TerritoryDescription</summary>
	Public Property TerritoryDescription() As System.String
		Get
			Return Me._TerritoryDescription
		End Get
		Set
			Me.IsSet_TerritoryDescription = True
			Me._TerritoryDescription = value
		End Set
	End Property
	''' <summary>設定フラグ：RegionID</summary>
	Public IsSet_RegionID As Boolean = False

	''' <summary>メンバ変数：RegionID</summary>
	Private _RegionID As Nullable(Of System.Int32)

	''' <summary>プロパティ：RegionID</summary>
	Public Property RegionID() As Nullable(Of System.Int32)
		Get
			Return Me._RegionID
		End Get
		Set
			Me.IsSet_RegionID = True
			Me._RegionID = value
		End Set
	End Property

	''' <summary>設定フラグ：Set_TerritoryID_forUPD</summary>
	Public IsSet_Set_TerritoryID_forUPD As Boolean = False

	''' <summary>メンバ変数：Set_TerritoryID_forUPD</summary>
	Private _Set_TerritoryID_forUPD As System.String

	''' <summary>プロパティ：Set_TerritoryID_forUPD</summary>
	Public Property Set_TerritoryID_forUPD() As System.String
		Get
			Return Me._Set_TerritoryID_forUPD
		End Get
		Set
			Me.IsSet_Set_TerritoryID_forUPD = True
			Me._Set_TerritoryID_forUPD = value
		End Set
	End Property
	''' <summary>設定フラグ：Set_TerritoryDescription_forUPD</summary>
	Public IsSet_Set_TerritoryDescription_forUPD As Boolean = False

	''' <summary>メンバ変数：Set_TerritoryDescription_forUPD</summary>
	Private _Set_TerritoryDescription_forUPD As System.String

	''' <summary>プロパティ：Set_TerritoryDescription_forUPD</summary>
	Public Property Set_TerritoryDescription_forUPD() As System.String
		Get
			Return Me._Set_TerritoryDescription_forUPD
		End Get
		Set
			Me.IsSet_Set_TerritoryDescription_forUPD = True
			Me._Set_TerritoryDescription_forUPD = value
		End Set
	End Property
	''' <summary>設定フラグ：Set_RegionID_forUPD</summary>
	Public IsSet_Set_RegionID_forUPD As Boolean = False

	''' <summary>メンバ変数：Set_RegionID_forUPD</summary>
	Private _Set_RegionID_forUPD As Nullable(Of System.Int32)

	''' <summary>プロパティ：Set_RegionID_forUPD</summary>
	Public Property Set_RegionID_forUPD() As Nullable(Of System.Int32)
		Get
			Return Me._Set_RegionID_forUPD
		End Get
		Set
			Me.IsSet_Set_RegionID_forUPD = True
			Me._Set_RegionID_forUPD = value
		End Set
	End Property

	''' <summary>設定フラグ：TerritoryID_Like</summary>
	Public IsSet_TerritoryID_Like As Boolean = False

	''' <summary>メンバ変数：TerritoryID_Like</summary>
	Private _TerritoryID_Like As System.String

	''' <summary>プロパティ：TerritoryID_Like</summary>
	Public Property TerritoryID_Like() As System.String
		Get
			Return Me._TerritoryID_Like
		End Get
		Set
			Me.IsSet_TerritoryID_Like = True
			Me._TerritoryID_Like = value
		End Set
	End Property
	''' <summary>設定フラグ：TerritoryDescription_Like</summary>
	Public IsSet_TerritoryDescription_Like As Boolean = False

	''' <summary>メンバ変数：TerritoryDescription_Like</summary>
	Private _TerritoryDescription_Like As System.String

	''' <summary>プロパティ：TerritoryDescription_Like</summary>
	Public Property TerritoryDescription_Like() As System.String
		Get
			Return Me._TerritoryDescription_Like
		End Get
		Set
			Me.IsSet_TerritoryDescription_Like = True
			Me._TerritoryDescription_Like = value
		End Set
	End Property
	''' <summary>設定フラグ：RegionID_Like</summary>
	Public IsSet_RegionID_Like As Boolean = False

	''' <summary>メンバ変数：RegionID_Like</summary>
	Private _RegionID_Like As Nullable(Of System.Int32)

	''' <summary>プロパティ：RegionID_Like</summary>
	Public Property RegionID_Like() As Nullable(Of System.Int32)
		Get
			Return Me._RegionID_Like
		End Get
		Set
			Me.IsSet_RegionID_Like = True
			Me._RegionID_Like = value
		End Set
	End Property

	#End Region
End Class

File: root/files/resource/Sql/sqlserver/DaoAutoGen/DTO/TerritoriesEntity.cs
//**********************************************************************************
//* クラス名        ：TerritoriesEntity
//* クラス日本語名  ：自動生成Entityクラス
//*
//* 作成日時        ：2014/2/9
//* 作成者          ：棟梁 D層自動生成ツール（墨壺）, 日立 太郎
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//**********************************************************************************

// System～
using System;

/// <summary>自動生成Entityクラス</summary>
[Serializable()]
public class TerritoriesEntity
{
    #region メンバ変数

    /// <summary>設定フラグ：TerritoryID</summary>
    public bool IsSetPK_TerritoryID = false;

    /// <summary>メンバ変数：TerritoryID</summary>
    private System.String _PK_TerritoryID;

    /// <summary>プロパティ：TerritoryID</summary>
    public System.String PK_TerritoryID
    {
        get
        {
            return this._PK_TerritoryID;
        }
        set
        {
            this.IsSetPK_TerritoryID = true;
            this._PK_TerritoryID = value;
        }
    }

    /// <summary>設定フラグ：TerritoryDescription</summary>
    public bool IsSet_TerritoryDescription = false;

    /// <summary>メンバ変数：TerritoryDescription</summary>
    private System.String _TerritoryDescription;

    /// <summary>プロパティ：TerritoryDescription</summary>
    public System.String TerritoryDescription
    {
        get
        {
            return this._TerritoryDescription;
        }
        set
        {
            this.IsSet_TerritoryDescription = true;
            this._TerritoryDescription = value;
        }
    }
    /// <summary>設定フラグ：RegionID</summary>
    public bool IsSet_RegionID = false;

    /// <summary>メンバ変数：RegionID</summary>
    private System.Int32? _RegionID;

    /// <summary>プロパティ：RegionID</summary>
    public System.Int32? RegionID
    {
        get
        {
            return this._RegionID;
        }
        set
        {
            this.IsSet_RegionID = true;
            this._RegionID = value;
        }
    }

    /// <summary>設定フラグ：Set_TerritoryID_forUPD</summary>
    public bool IsSet_Set_TerritoryID_forUPD = false;

    /// <summary>メンバ変数：Set_TerritoryID_forUPD</summary>
    private System.String _Set_TerritoryID_forUPD;

    /// <summary>プロパティ：Set_TerritoryID_forUPD</summary>
    public System.String Set_TerritoryID_forUPD
    {
        get
        {
            return this._Set_TerritoryID_forUPD;
        }
        set
        {
            this.IsSet_Set_TerritoryID_forUPD = true;
            this._Set_TerritoryID_forUPD = value;
        }
    }
    /// <summary>設定フラグ：Set_TerritoryDescription_forUPD</summary>
    public bool IsSet_Set_TerritoryDescription_forUPD = false;

    /// <summary>メンバ変数：Set_TerritoryDescription_forUPD</summary>
    private System.String _Set_TerritoryDescription_forUPD;

    /// <summary>プロパティ：Set_TerritoryDescription_forUPD</summary>
    public System.String Set_TerritoryDescription_forUPD
    {
        get
        {
            return this._Set_TerritoryDescription_forUPD;
        }
        set
        {
            this.IsSet_Set_TerritoryDescription_forUPD = true;
            this._Set_TerritoryDescription_forUPD = value;
        }
    }
    /// <summary>設定フラグ：Set_RegionID_forUPD</summary>
    public bool IsSet_Set_RegionID_forUPD = false;

    /// <summary>メンバ変数：Set_RegionID_forUPD</summary>
    private System.Int32? _Set_RegionID_forUPD;

    /// <summary>プロパティ：Set_RegionID_forUPD</summary>
    public System.Int32? Set_RegionID_forUPD
    {
        get
        {
            return this._Set_RegionID_forUPD;
        }
        set
        {
            this.IsSet_Set_RegionID_forUPD = true;
            this._Set_RegionID_forUPD = value;
        }
    }

    /// <summary>設定フラグ：TerritoryID_Like</summary>
    public bool IsSet_TerritoryID_Like = false;

    /// <summary>メンバ変数：TerritoryID_Like</summary>
    private System.String _TerritoryID_Like;

    /// <summary>プロパティ：TerritoryID_Like</summary>
    public System.String TerritoryID_Like
    {
        get
        {
            return this._TerritoryID_Like;
        }
        set
        {
            this.IsSet_TerritoryID_Like = true;
            this._TerritoryID_Like = value;
        }
    }
    /// <summary>設定フラグ：TerritoryDescription_Like</summary>
    public bool IsSet_TerritoryDescription_Like = false;

    /// <summary>メンバ変数：TerritoryDescription_Like</summary>
    private System.String _TerritoryDescription_Like;

    /// <summary>プロパティ：TerritoryDescription_Like</summary>
    public System.String TerritoryDescription_Like
    {
        get
        {
            return this._TerritoryDescription_Like;
        }
        set
        {
            this.IsSet_TerritoryDescription_Like = true;
            this._TerritoryDescription_Like = value;
        }
    }
    /// <summary>設定フラグ：RegionID_Like</summary>
    public bool IsSet_RegionID_Like = false;

    /// <summary>メンバ変数：RegionID_Like</summary>
    private System.Int32? _RegionID_Like;

    /// <summary>プロパティ：RegionID_Like</summary>
    public System.Int32? RegionID_Like
    {
        get
        {
            return this._RegionID_Like;
        }
        set
        {
            this.IsSet_RegionID_Like = true;
            this._RegionID_Like = value;
        }
    }

    #endregion
}
